fusor-ember-cli/app/templates/cloudforms/cfme-configuration.hbs

```diff
@@ -5,14 +5,14 @@
                cssId="cfme_root_password"
                isRequired=true
                disabled=isStarted
-               minChars=8
+               validator=passwordValidator
```

Comment:
so, the password validation is a convenience validator that would enforce our password policies that we want to.  So if we want to enforce complexity or something else, we could do it there.  Custom passwords (like overcloud password) would use their own custom validators (such as just length)